sklearn/tests/test_kernel_ridge.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+from sklearn.datasets import make_classification
+from sklearn.linear_model import Ridge
+from sklearn.kernel_ridge import KernelRidge
+
+from sklearn.utils.testing import assert_array_almost_equal
+
+
+X, y = make_classification(n_classes=2, random_state=0)
+Y = np.array([y, y]).T
+
+
+def test_kernel_ridge():
+    pred = Ridge(alpha=1, fit_intercept=False).fit(X, y).predict(X)
+    pred2 = KernelRidge(kernel="linear", alpha=1).fit(X, y).predict(X)
+    assert_array_almost_equal(pred, pred2)
+
+
+def test_kernel_ridge_precomputed():
+    K = np.dot(X, X.T)
+    pred = KernelRidge(kernel="linear").fit(X, y).predict(X)
```

Comment:
I'd also prefer decision function here. predict is a very rough approximation of "the same mode".